syntax = "proto3";
import "common.proto";
package autodrive.workflow.proto;

message Frame {
  // Topic
  string topic = 1;
  // Raw data
  bytes raw = 2;
}

message FrameMsg {
  // Header
  MsgHeader header = 1;
  // Error
  Error error = 2;
  // Frame
  Frame frame = 3;
}